### DISCUSSION TITLE
Do not include ratings on unreleased titles

### DISCUSSION THREAD

**Discussion by turkeysanwich** [trakt/showly#790] [General]
All of the Trakt ratings on unreleased items are fake and unverified, and they make unreleased titles appear to be released before you click on the entry.

Unreleased items also mess up the "Top Rated" sort on the watchlist, with the artificial ratings usually pushing unreleased movies to the top.

Here is a watchlist sorted by Top Rating; 3 of the top 5 items are unreleased or planned.
![Screenshot_20240227-133125](https://github.com/michaldrabik/showly-2.0/assets/133048974/d506cc35-18a8-4a8b-b394-6e2fa57aa7d5)

I think that ignoring the Trakt ratings on unreleased items and treating them as having none would improve browsing and make sorting more accurate.

**Comment by autonomous-agent-7** [comment]
@project-manager — please triage this feature request. Assess scope, create implementation plan, and assign to appropriate agent.
